### PR TITLE
Add support for binding predictor configuration

### PR DESCRIPTION
**Pull request by ihodes**
Also generalize the Java heap config

### CODE REVIEW

File: src/lib/parameters.ml [hunk 1]
```diff
@@ -16,16 +16,16 @@ type t = {
   (** The reference build *)
   results_path: string;
   (** Where to save the results. *)
-  picard_java_max_heap: string option;
-  (** Max Java heap size used for Picard tools e.g. 8g, 256m. *)
+  java_max_heap: string option;
+  (** Max Java heap size used for Picard & GAT tools e.g. 8g, 256m. *)
```
Comment (smondet) [inline src/lib/parameters.ml]: typo

File: src/lib/parameters.ml [hunk 2]
```diff
@@ -16,16 +16,16 @@ type t = {
...
   igv_url_server_prefix: string option [@env "IGV_URL_SERVER_PREFIX"];
   (** URL with which to prefix igvxml paths. *)
   realign_bams: bool [@default true]
       [@name "without-realigning-bams"];
   (** Don't realign input BAMs. *)
   use_bwa_mem_opt: bool [@default true]
       [@name "without-bwa-mem-optimized"];
-  (** Don't use the optimized workflow-node for bwa-mem \
+  (** Don't puse the optimized workflow-node for bwa-mem \
```
Comment (smondet) [inline src/lib/parameters.ml]: typo

**Review by smondet — COMMENTED** [review]
:+1: **`mod`** typos

**Comment by ihodes** [comment]
🙏  Typos fixed; waiting on Biokepi PR https://github.com/hammerlab/biokepi/pull/453

**Pull request merged**